### PR TITLE
Use include_str! 

### PR DESCRIPTION
**Pull request by ketkarameya**
Since we know the path to the language specific rules/edges/scopes at static time, we can use the `include_str!` feature.

### CODE REVIEW

File: polyglot/piranha/src/config.rs [hunk 1]
```diff
@@ -22,10 +22,10 @@ use crate::{
     rule::{Rule, Rules},
     scopes::{ScopeConfig, ScopeGenerator},
   },
-  utilities::read_toml,
+  utilities::{read_toml, parse_toml},
 };
 
-use std::path::{Path, PathBuf};
+use std::path::{Path};
```
Comment (lazaroclapp) [inline polyglot/piranha/src/config.rs:28]: `use std::path::Path;

File: polyglot/piranha/src/utilities/mod.rs [hunk 2]
```diff
@@ -92,6 +99,9 @@ pub(crate) fn find_file(input_dir: &PathBuf, name: &str) -> PathBuf {
     .path()
 }
 
+
+
+
```
Comment (lazaroclapp) [inline polyglot/piranha/src/utilities/mod.rs:104]: What are these newlines for?

**Review by lazaroclapp — APPROVED** [review]
👍

File: polyglot/piranha/src/utilities/mod.rs (hunk 2 above)
Comment (ketkarameya) [reply polyglot/piranha/src/utilities/mod.rs:104]: I Am going to see how to run cargo fmt each time before the code gets to review.

Comment (lazaroclapp) [reply polyglot/piranha/src/utilities/mod.rs:104]: Have you considered setting it up as a git pre-commit hook and adding a CI job to check for it on PRs?

**Pull request merged**